src/Functions/array/arrayUniq.cpp

```diff
@@ -317,10 +317,46 @@ void FunctionArrayUniq::executeHashed(
     executeMethod<MethodHashed>(offsets, columns, {}, nullptr, res_values);
 }
 
-
 REGISTER_FUNCTION(ArrayUniq)
 {
-    factory.registerFunction<FunctionArrayUniq>();
+    FunctionDocumentation::Description description = R"(
+For a single argument passed, counts the number of different elements in the array.
+For multiple arguments passed, it counts the number of different **tuples** made of elements at matching positions across multiple arrays.
+
+For example `SELECT arrayUniq([1,2], [3,4], [5,6])` will form the following tuples:
+* Position 1: (1,3,5)
+* Position 2: (2,4,6)
+
+It will then count the number of unique tuples. In this case `2`.
+
+:::tip
+If you want to get a list of unique items in an array, you can use `arrayReduce('groupUniqArray', arr)`.
+:::
+)";
+    FunctionDocumentation::Syntax syntax = "arrayUniq(x[, ...yN])";
+    FunctionDocumentation::Arguments arguments = {
+        {
+            "x",
+            "Array for which to count the number of unique elements. [`Array`](/sql-reference/data-types/array)."
+        },
+        {
+            "...yN (optional)",
+            "Additional same-size arrays used to count the number of unique tuples of elements at corresponding positions in multiple arrays. [`Array`](/sql-reference/data-types/array)."
+        }
+    };
+    FunctionDocumentation::Examples examples =
+{{"Single argument", "SELECT arrayUniq([1,1,2,2])", "2"},
+{"Multiple argument", "SELECT arrayUniq([1,2,3], [4,5,6])", "3"}};
+    FunctionDocumentation::ReturnedValue returned_value = R"(
+For a single argument returns the number of unique
+elements. For multiple arguments returns the number of unique tuples made from
+elements at corresponding positions across the arrays.
+[`UInt32`](/sql-reference/data-types/int-uint).
+)";
+    FunctionDocumentation::IntroducedIn introduced_in = {1, 1};
```

Comment:
Same comment as in your other PR ... is it really version 1.1?

Reply:
@rschu1ze I've been going off of the list Alexey provided but maybe it has some inaccuracies. Will cross check with changelogs.

Reply:
Nope, Alexey's list is the authorative source and if it says 1.1, then be it so.
Let's not waste time for reading changelogs. For 99.9% of all users it won't matter if a version was introduced with 1.1 or v18.4. Only the last 3 versions + some LTS versions are supported, so old versions wouldn't matter anyways.